saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TwitterSentimentAnalysis/README.md

```diff
@@ -0,0 +1,75 @@
+
+# illinois-twitter-client
```

Comment:
I am not sure how this readme is relevant to this task. It just explains how to use someone else's twitter client, it doesn't say anything on the application. Maybe revise this readme with more relevant details? Like what is the input, what is the output, how to run, etc.